acme.sh

```diff
@@ -6790,7 +6793,7 @@ _process() {
 
   if [ "${_CMD}" != "install" ]; then
     if [ "$__INTERACTIVE" ] && ! _checkSudo; then
-      if [ -z "$FORCE" ]; then
+      if [ -z "$FORCE" -a -z "$ALLOW_SUDO" ]; then
```

Comment:
Hello @bjmgeek, I want to thank you for opening this PR, as I have the same need.

I noticed one style nit, that `-a` in Bash is a deprecated style, due to some syntax ambiguity; what is commonly preferred is `&&`, like the following:

```suggestion
      if [ -z "$FORCE" ] && [ -z "$ALLOW_SUDO" ]; then
```